src/commands/kv/rename_namespace.rs

```diff
@@ -0,0 +1,26 @@
+use cloudflare::apiclient::APIClient;
+
+use cloudflare::workerskv::rename_namespace::RenameNamespace;
+use cloudflare::workerskv::rename_namespace::RenameNamespaceParams;
+
+use crate::terminal::message;
+
+pub fn rename_namespace(id: &str, title: &str) -> Result<(), failure::Error> {
+    let client = super::api_client()?;
+    let account_id = super::account_id()?;
+
+    let msg = format!("Renaming namespace {} with title \"{}\"", id, title);
```

Comment:
So, this is a minor thing, but I read this message as "renaming a namespace with this title" and then wonder "rename to what?"

What about something like

> Renaming namespace ID to have title TITLE

or

> Renaming namespace ID; new title is TITLE

or something like that?